java/java.editor.base/test/unit/src/org/netbeans/modules/java/editor/base/javadoc/JavadocImportsTest.java

```diff
@@ -207,6 +207,122 @@ public void testComputeReferencedElements() throws Exception {
         assertEquals(exp, sortedResult);
     }
     
+    public void testComputeReferencedElementsMarkdown() throws Exception {
+        String code =
+                "package p;\n" +
+                "import java.io.IOException;\n" +
```

Comment:
sidenote: bumping the lang level for multi line Strings could make it easier to write new tests with code snippets.